pkg/controllers/apps/description/description.go

```diff
@@ -366,3 +368,10 @@ func (c *Controller) enqueue(desc *appsapi.Description) {
 	}
 	c.workqueue.Add(key)
 }
+
+func (c *Controller) Enqueue(obj runtime.Object) {
```

Comment:
I don't think we need to enqueue a whole `Description`, which declares multiple resources.

Reply:
Right, I created a generic Resource Controller with individual queue, whole Description enqueue will not happen.